graylog2-server/src/main/java/org/graylog2/indexer/cluster/Cluster.java

```diff
@@ -176,12 +217,18 @@ public void waitForConnectedAndHealthy(long timeout, TimeUnit unit) throws Inter
         scheduledFuture.cancel(true); // Make sure to cancel the task to avoid task leaks!
 
         if (!waitSuccess) {
-            throw new TimeoutException("Elasticsearch cluster didn't get healthy within timeout");
+            throw new TimeoutException("Elasticsearch cluster or write index didn't get healthy within timeout");
```

Comment:
I would remove the part about the cluster completely: "Write-active index didn't get healthy within timeout."